### PR TITLE
Fix max & min items is repeater when using groups

### PR DESCRIPTION
**Pull request by c2webstudio**
Updated repeater when using the grouped functionality after adding the item was not removing the add more prompt if max-Items where reached.

### CODE REVIEW

**Comment by LukeTowers** [comment]
I think the real issue is that https://github.com/octobercms/october/blob/master/modules/backend/formwidgets/repeater/assets/js/repeater.js#L45 is not getting fired in your specific case. Could you look into why that is @c2webstudio?

**Comment by c2webstudio** [comment]
Yes @LukeTowers  the https://github.com/octobercms/october/blob/master/modules/backend/formwidgets/repeater/assets/js/repeater.js#L45 is not fired as in my case i am adding a group that fires the https://github.com/octobercms/october/blob/master/modules/backend/formwidgets/repeater/assets/js/repeater.js#L47

File: modules/backend/formwidgets/repeater/assets/js/repeater.js [hunk 1]
```diff
@@ -84,6 +84,7 @@
     }
 
     Repeater.prototype.clickAddGroupButton = function(ev) {
+        var  $self = this;
```
Comment (LukeTowers) [inline modules/backend/formwidgets/repeater/assets/js/repeater.js]: One too many spaces between `var` and `$self`

**Comment by LukeTowers** [comment]
@c2webstudio fair enough :) added a comment, fix that and I'll merge it

**Pull request merged**